### DISCUSSION TITLE
What's the usage or effect of `meta.json` in the base demo project of `turborepo`?

### DISCUSSION THREAD

**Discussion by gyhyfj** [vercel/turborepo#5542] [Help] [answered]
The demo link is https://github.com/vercel/turbo/tree/main/examples/basic ,
I found `meta.json` in the root of this project:
```json
{
  "name": "Next.js",
  "description": "Minimal Turborepo example for learning the fundamentals.",
  "template": "https://vercel.com/templates/next.js/turborepo-next-basic",
  "featured": true,
  "boost": true
}
```
What's the usage or effect ofWhat's the usage or effect of it?

**Comment by mehulkar** [comment, marked as answer]
We use this to populate this page https://turbo.build/repo/docs/getting-started/from-example. You do not need to use it if you're using the example as a starterkit!
